### ISSUE TITLE
Improve harvester to check DOI URLs

### ISSUE THREAD

**Issue by steingod** [steingod/mdharvest#17]
Some data providers do not add the full DOI URL to the exposed discovery metadata. The catalogue interface (https://github.com/metno/metsis-drupal) require this to be a URL. Add support in the harvester to add https:// to DOIs missing this.

**Comment by ferrighi** [comment]
I think drupal now handles both. But anyway, should this be done with xslt? I understand that for DIF 10 at least it is correct to provide the value of the DOI through: 
```
  <Persistent_Identifier>
    <Type>DOI</Type>
    <Identifier>10.5067/IAGYM8Q26QRE</Identifier>
    <Authority>https://doi.org/</Authority>
  </Persistent_Identifier>
```
so we should handle this in the translation to mmd. Or was this related to other parts of the harvester?

**Comment by steingod** [comment]
No what you suggest makes sense to me. That would simplify and also be consistent in the manner that what we get is actually taken to disk and then we interpret and translate. Will you create an issue for the XSLT? Should be handled by all XSLT's.

**Comment by ferrighi** [comment]
Yes it should be handled properly both in XSLT and in Drupal now. Closing.
